xc7/README.rst

```diff
@@ -88,14 +88,11 @@ To build the litex example, run the following commands:
 .. code:: bash
         :name: xc7-litex
 
-        wget https://raw.githubusercontent.com/enjoy-digital/litex/master/litex_setup.py
-        chmod +x litex_setup.py
-        ./litex_setup.py init
-        ./litex_setup.py install
         wget https://static.dev.sifive.com/dev-tools/riscv64-unknown-elf-gcc-8.1.0-2019.01.0-x86_64-linux-ubuntu14.tar.gz
         tar -xf riscv64-unknown-elf-gcc-8.1.0-2019.01.0-x86_64-linux-ubuntu14.tar.gz
         export PATH=$PATH:$PWD/riscv64-unknown-elf-gcc-8.1.0-2019.01.0-x86_64-linux-ubuntu14/bin/
-        pushd litex/litex/boards/targets && ./arty.py --toolchain symbiflow --cpu-type vexriscv --build && popd
+        pushd xc7/litex_demo && make clean && TARGET="arty_50" make && popd
```

Comment:
These should probably be separate command blocks.

Reply:
All the examples have instructions for bitstream generation for various boards in the same block.

Reply:
Probably should have seperate blocks for different boards so they can be tested separately and also appear in separate tabs?

Reply:
Yes, we are adding this feature for the Sphinx-based version of the symbiflow-examples documentation:
![docs](https://user-images.githubusercontent.com/52699314/100758430-cb8abe80-33ef-11eb-83e3-7929efa52231.png)

However, having separate block commands is not consistent with the rest of the current repository and will cause that the main README will become less readable (because the whole LiteX readme will be merged with the main README). We are also thinking about a smarter mechanism for local READMEs in each example directory collected then by the main documentation. We have some initial implementation for this case. 

I'm wondering if we can split the changes from this PR into two parts. We will avoid duplicating the work. The first part will add an example code with the main README update (in the old way, as it is done now). 

The second part with the local README for this example will be added together with the rest of the sphinx-based documentation. I believe that soon the PR for the sphinx-based version will be created.

Reply:
Probably we can prepare a WIP PR with the sphinx-based version, so you will be sure that all the changes, related to this example are there.